### DISCUSSION TITLE
Integrity constraint violation on creating role with team id

### DISCUSSION THREAD

**Discussion by Jaydeep7248** [spatie/laravel-permission#1968] [Q&A] [answered]
When I try to create role with same name but different team id, it doesn't allow to do it and throws the error.

1062 - Duplicate entry 'admin-web' for key 'roles_name_guard_name_unique'

Can you please help in the same?

Thanks.

**Comment by erikn69** [comment, marked as answer]
Duplicate of #1966, answered on https://github.com/spatie/laravel-permission/issues/1966#issuecomment-998800768
